libraries/renderer/include/renderer.hpp

```diff
@@ -2,7 +2,7 @@
 
 #include <bnb/types/full_image.hpp>
 
-#include "program.hpp"
```

Comment:
looks like this include (and corresponding .cpp) is not required anymore (were moved somewhere), but such files still present in this example